sensor_msgs/LaserScan scan
nav_msgs/Path path
int8[] local_map
int8[] agents_map
int8[] neighbors_goal_map
int8[] planner_map